rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -141,135 +149,136 @@ type CollateralTaxonomy: <"Specifies the collateral taxonomy, which is composed
+        then regionalGovernmentType is absent
 
-	condition SpecialPurposeVehicleSubType:
-		if issuerType <> IssuerTypeEnum -> SpecialPurposeVehicle
-		then specialPurposeVehicleType is absent
+    condition SpecialPurposeVehicleSubType:
+        if issuerType <> IssuerTypeEnum -> SpecialPurposeVehicle
+        then specialPurposeVehicleType is absent
 
 type QuasiGovernmentIssuerType: <"Represents a class to allow specification of different types of Quasi Government collateral.">
 
-	sovereignEntity boolean (1..1) <"True if sovereign entity (e.g. not separate legal personality from sovereign) or false if non-sovereign entity (e.g. separate legal personality from sovereign).">
-	sovereignRecourse boolean (0..1) <"Applies to non-sovereign entity (e.g. separate legal personality from sovereign).  True if entity has recourse to sovereign (e.g. debt guaranteed by government).  False if entity does not have recourse to sovereign.">
-	
-	condition NonSovereignEntityRecourse:
-		if sovereignRecourse exists then sovereignEntity = False
+    sovereignEntity boolean (1..1) <"True if sovereign entity (e.g. not separate legal personality from sovereign) or false if non-sovereign entity (e.g. separate legal personality from sovereign).">
+    sovereignRecourse boolean (0..1) <"Applies to non-sovereign entity (e.g. separate legal personality from sovereign).  True if entity has recourse to sovereign (e.g. debt guaranteed by government).  False if entity does not have recourse to sovereign.">
+
+    condition NonSovereignEntityRecourse:
+        if sovereignRecourse exists then sovereignEntity = False
 
 type RegionalGovernmentIssuerType: <"Represents a class to allow specification of different type of Regional government collateral.">
 
-	sovereignRecourse boolean (1..1) <"Applies to regional governments, local authorities or municipals.  True if entity has recourse to sovereign (e.g. debt guaranteed by government).  False if entity does not have recourse to sovereign.">
-	
+    sovereignRecourse boolean (1..1) <"Applies to regional governments, local authorities or municipals.  True if entity has recourse to sovereign (e.g. debt guaranteed by government).  False if entity does not have recourse to sovereign.">
+
 type SpecialPurposeVehicleIssuerType: <"Represents a class to allow specification of different types of special purpose vehicle (SPV) collateral.">
-	creditRisk CreditRiskEnum (0..1) <"Indicates tranched or untranched credit risk."> 
+    creditRisk CreditRiskEnum (0..1) <"Indicates tranched or untranched credit risk.">
 
 type TaxonomyClassification:
-	className string (1..1) <"The name defined by the classification system for a specific attribute in the taxonomy.">
-	value string (1..1) <"The value set by the taxonomu that is specific to the className attribute.">
-	description string (0..1) <"A descuription of the class.">
+    className string (0..1) <"The name defined by the classification system for a specific attribute in the taxonomy">
```

Comment:
Could you give a bit more detail as to why the cardinality of className has been relaxed please?